### PR TITLE
Added subscription details to billing

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?
- Added a Plan Details sections
- Reorganized the docs, moved account management information out of Invoiced to Plan details
- Added subscription details to Plan details
- Rename Invoices to Payment

### Motivation
https://datadoghq.atlassian.net/browse/DOCS-4977

### Additional Notes
Ready to merge after review!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/billing/_index.md [hunk 1]
```diff
@@ -33,43 +33,53 @@ Datadog meters the count of IoT devices hourly. The billable count of IoT device
...
+{{< img src="account_management/billing/BillingContactDetailsOverview.png" alt="Billing contact details on the Plan page" style="width:90%;" >}}
 
-{{< img src="account_management/billing/PaymentMethodOverview.png" alt="Payment method on the Plan page" style="width:80%;" >}}
+Through **Edit Details**, you can add, edit or remove your billing address. You can also set specific email addresses to receive invoices.
 
-**Edit Payment** provides options to manage payment methods. You can edit or remove cards, and request to change your payment method from card to invoice and vice versa. 
+{{< img src="account_management/billing/BillingContactDetailsEdit.png" alt="Editing billing contact details on the Plan page" style="width:90%;" >}}
 
-{{< img src="account_management/billing/PaymentSettingsDetails.png" alt="Payment settings on the Plan page" style="width:80%;" >}}
+**Note**: The email address does not need to be a team member within Datadog. For example, you could use `invoices@yourcompany.com`.
```
Comment (drichards-87) [inline content/en/account_management/billing/_index.md:58]: Just noting that as a general rule of thumb, we should be cautious when creating fictitious names (or in this case, an email address). For email addresses, the domain @example.com is specifically reserved for examples.

File: content/en/account_management/billing/_index.md [hunk 2]
```diff
@@ -33,43 +33,53 @@ Datadog meters the count of IoT devices hourly. The billable count of IoT device
...
 
-### Managing your billing contact details
+### View your subscription details
 
-You can view your billing contact details on the [**Billing Contact Details**][12] section. 
+The [Subscription Details][8] section includes the quantity, contract price and on-demand price for all committed products.
```
Comment (drichards-87) [inline content/en/account_management/billing/_index.md:62]:
```suggestion
The [Subscription Details][8] section includes the quantity, contract price, and on-demand price for all committed products.
```

File: content/en/account_management/billing/_index.md [hunk 3]
```diff
@@ -33,43 +33,53 @@ Datadog meters the count of IoT devices hourly. The billable count of IoT device
...
+### Managing your billing contact details
 
-### Managing your payment method
+You can view your billing contact details on the [**Billing Contact Details**][8] section. 
 
-The [**Payment Method**][12] section contains details on your payment options. 
+{{< img src="account_management/billing/BillingContactDetailsOverview.png" alt="Billing contact details on the Plan page" style="width:90%;" >}}
 
-{{< img src="account_management/billing/PaymentMethodOverview.png" alt="Payment method on the Plan page" style="width:80%;" >}}
+Through **Edit Details**, you can add, edit or remove your billing address. You can also set specific email addresses to receive invoices.
```
Comment (drichards-87) [inline content/en/account_management/billing/_index.md:54]:
```suggestion
**Edit Details** to add, edit, or remove your billing address. You can also specify the email addresses where invoices should be sent.
```

File: content/en/account_management/billing/_index.md [hunk 4]
```diff
@@ -33,43 +33,53 @@ Datadog meters the count of IoT devices hourly. The billable count of IoT device
...
+
+### Credit card
+
+If you pay by credit card, receipts are available to [Administrators][9] for previous months under [Billing History][10]. For copies of your invoice, email [Datadog billing][11].
+
+See [Credit Card Billing][12] for more details.
+
+### Invoicing
+
+If you pay by check, ACH, or wire, invoices are emailed to the billing email addresses near the 10th business day of each month. If you need an additional copy, email [Datadog billing][11]. Details on where to remit payment can be found on the invoice.
```
Comment (drichards-87) [inline content/en/account_management/billing/_index.md:82]:
```suggestion
If you pay by check, ACH, or wire, invoices are emailed to the billing email addresses near the 10th business day of each month. To request an additional copy, email [Datadog billing][11]. Details on where to remit payment can be found on the invoice.
```

Comment (drichards-87) [reply content/en/account_management/billing/_index.md:82]: Re: where we say "near the 10th business day of each month..." I'm wondering if there's a way for us to be more specific. Do you happen to know how this works? Is there a date range we can specify here?

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Looks good overall. I just had a couple of minor pieces of feedback.

File: content/en/account_management/billing/_index.md (hunk 4 above)
Comment (estherk15) [reply content/en/account_management/billing/_index.md:82]: No, I'm not entirely sure on this, but I could ask!

**Review by drichards-87 — APPROVED** [review]
Looks good!

**Pull request merged**